src/meta.ts

```diff
@@ -39,9 +41,16 @@ export class Meta {
     this.tags = tcl.Transform(inputs.tags);
     this.flavor = fcl.Transform(inputs.flavor);
     this.date = new Date();
+    this.commit_date = this.getCommitDate();
     this.version = this.getVersion();
   }
 
+  private getCommitDate(): Date {
+    const myOutput = execSync('git show -s --format="%ci" HEAD');
```

Comment:
We should use our toolkit for this. Can you open a PR to add this in https://github.com/docker/actions-toolkit/blob/main/src/git.ts?

Reply:
Also we should only invoke git command if `context` is `git` like we do in https://github.com/docker/metadata-action/blob/35e9aff4f5d665b5aa8a8f2adffaf8a1b5f49cc0/src/context.ts#L39-L43

If context is `workflow` then we should take it from GH API if feasible. `commits.timestamp` looks to be a good candidate: https://github.com/docker/metadata-action/blob/35e9aff4f5d665b5aa8a8f2adffaf8a1b5f49cc0/__tests__/fixtures/event_push_master.json#L20

Reply:
> add a new property to `GitContext` looks complex...

Just smth similar to https://github.com/docker/actions-toolkit/blob/f1593e3aa24d872bd4fa0a8f76a3bdaf26d20872/src/git.ts#L101-L103 is enough.

Reply:
> add a new property to `GitContext` looks complex...

`GitContext` is just actuall `GitHubContext` and it's created from `Context.constructor`, make it very hard to add a new property, should I create some new interface types to resolve this?

Reply:
I mean a new method for `Git` class in the actions toolkit like:

```
public static async commitDate(): Promise<Date> {
```

Then you would just need to consume it here when `context` is `git`.